website/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

```diff
@@ -53,9 +53,8 @@ If an organization's owners disable SSO (or downgrade the organization's account
 
    ![Screenshot: Terraform Cloud's create account page.](/img/docs/create-account.png)
 
-   ![Screenshot: The password prompt for linking an SSO email address to the currently signed-in Terraform Cloud account.](/img/docs/matched-linking.png)
-
-   ![Screenshot: the username and password prompt for linking an SSO email address to another Terraform Cloud account.](/img/docs/chosen-linking.png)
+    TODO: add new screen shot once PR is deployed https://github.com/hashicorp/atlas/pull/13220
```

Comment:
Will update with screenshot from production using Safari so that it looks like the other screenshots on this page. For reference it will look something like this: 
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/11843680/186768294-faf9959c-c218-49c7-90c8-1274c9345394.png">

Reply:
you know, I honestly wonder whether we NEED a screenshot here. Screenshots are so difficult to maintain over time. Often it's more than good enough to just tell users where to go and what they will see. Thoughts on this?   If you think we can do without, let's remove!

Reply:
I think we can definitely do without. Would we want to replace this with text or is it fine to just remove the screenshots from this section?